app/lambda/lambda_base.py

```diff
@@ -0,0 +1,13 @@
+"""Base class for implementing Lambda handlers as classes.
```

Comment:
Do we really need this class? Since we are implementing the handler in each lambda, I don't think this is required.